public/index.html

```diff
@@ -8,9 +8,46 @@
   <!--<link rel="stylesheet" href="/css/themes/fresh/app.css" />-->
   <link rel="stylesheet" href="/css/themes/helvetica/app.css" />
   <link rel="stylesheet" href="/css/font-awesome.min.css">
+  <noscript>
+    <style>
+      .jsonly { display: none }
+    </style>
+  </noscript>
   <link rel="icon" href="/img/favicon.png" type="image/x-icon" />
 </head>
 <body>
+  <div id="ember-app">
+    <div class="container">
+      <div class="row header-row">
+        <div class="col-md-4">
+          <div class="header">
+            <h1 class="title">
+              <a href="/">freefeed</a>
```

Comment:
This should be "Pepyatka", I suppose. "FreeFeed" labeling should go to `FreeFeed/pepyatka-html` repo.